### ISSUE TITLE
Append email regex to also include gmail addresses

### ISSUE THREAD

**Issue by MufasaTheMusician** [MufasaTheMusician/eatwithme#100]
The Android store only permits gmail accounts to participate in limited releases, so we will need to allow users to create an account using a gmail address.

The nature of the limited release will mean that only those invited will have access to download the app so the scope should still be limited to the ANU environment

**Comment by MufasaTheMusician** [comment]
Should be able to find the filter in the login page, just needs modification to allow "gmail.com" as well.

**Comment by MufasaTheMusician** [comment]
How is this coming Guang? Would be good to have this before Thursday so it can work with alpha tester gmail accounts

**Comment by Guang980803** [comment]
Sorry for the late reply, just had time to look at it. 
The Google play indeed needs tester's gmail address, but it needs to be manually typed in. 
Do you mean that users can use gmail to register for the app.

**Comment by MufasaTheMusician** [comment]
Yep, referring to opening the regex so that users can login to our app using a gmail address

**Comment by MufasaTheMusician** [comment]
Might open this up to allow any email address to open the possibilities of future testing for Masaba. Can still limit any betas through the stores if necessary but also allows it to be open to the public if going down that route later.
